packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -1671,6 +1672,123 @@ describe('Checkout', () => {
             });
         });
 
+        describe('lookupGeoLocation ::', () => {
+            afterEach(() => {
+                jest.clearAllMocks();
+            });
+
+            describe('when `getGeoLocation` request succeeds', () => {
+                // Arrange
+                let wrapper;
+                let getGeoLocationSpy;
+
+                beforeEach(async () => {
+                    jest.spyOn(VueCheckout.methods, 'initialise').mockImplementation();
+
+                    getGeoLocationSpy = jest.spyOn(VueCheckout.methods, 'getGeoLocation');
+
+                    wrapper = mount(VueCheckout, {
+                        store: createStore(),
+                        i18n,
+                        localVue,
+                        propsData,
+                        mocks: {
+                            $logger
+                        }
+                    });
+
+                    await flushPromises();
+                });
+
+                it('then it should not call `logWarn`', async () => {
```

Comment:
Could you change these so it starts with a `should`. e.g `it('should not call logWarn')`